### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Evan Rothen

### DISCUSSION THREAD

**Discussion by EVAN-ROTHEN** [heig-vd-dai-course/heig-vd-dai-course#288] [Show and tell]
[The link to your GitHub profile README!](https://github.com/EVAN-ROTHEN/EVAN-ROTHEN)

**Comment by ludelafo** [comment]
✨ _Special Java profile_ ✨

Jokes aside, nice profile, thank you!
